hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/ExponentialRetryPolicy.java

```diff
@@ -75,23 +58,15 @@ public class ExponentialRetryPolicy extends AbfsRetryPolicy {
    */
   private final int minBackoff;
 
-  /**
-   * Initializes a new instance of the {@link ExponentialRetryPolicy} class.
-   */
-  public ExponentialRetryPolicy(final int maxIoRetries) {
-
-    this(maxIoRetries, DEFAULT_MIN_BACKOFF, DEFAULT_MAX_BACKOFF,
-        DEFAULT_CLIENT_BACKOFF);
-  }
-
   /**
    * Initializes a new instance of the {@link ExponentialRetryPolicy} class.
    *
-   * @param conf The {@link AbfsConfiguration} from which to retrieve retry configuration.
+   *  @param conf    The {@link AbfsConfiguration} from which to retrieve retry configuration.
+   *   @param maxIoRetries The maximum number of retry attempts
    */
-  public ExponentialRetryPolicy(AbfsConfiguration conf) {
-    this(conf.getMaxIoRetries(), conf.getMinBackoffIntervalMilliseconds(), conf.getMaxBackoffIntervalMilliseconds(),
```

Comment:
This is the primary contructor used everywhere I guess, better we do not change it.